pydantic_ai_slim/pydantic_ai/models/openai.py

```diff
@@ -85,6 +85,12 @@ class OpenAIModelSettings(ModelSettings):
     result in faster responses and fewer tokens used on reasoning in a response.
     """
 
+    logprobs: bool
```

Comment:
Can you please prefix these with `openai_`? Note that the model settings classes for other models specifically state `ALL FIELDS MUST BE `anthropic_` PREFIXED SO YOU CAN MERGE THEM WITH OTHER MODELS.` in their docstring -- can you add the same here?

If these are supported by multiple models (worth checking, if you have a moment), they can go onto the main ModelSettings class.